source/base/DateTime.ooc

```diff
@@ -14,22 +14,251 @@
 * You should have received a copy of the GNU Lesser General Public License
 * along with this software. If not, see <http://www.gnu.org/licenses/>.
 */
-TimeSpan: cover {
-	_ticks: Int64
-	ticks ::= this _ticks
-	init: func@ (=_ticks)
-	kean_base_timeSpan_getTicks: unmangled func -> Int64 { this _ticks }
-	new: unmangled(kean_base_timeSpan_new) static func ~API (ticks: UInt64) -> This { This new(ticks) }
+
+use ooc-base
+import os/Time
+
+DateTimeData: class {
```

Comment:
Making it a 'cover' seems to be working, thanks.
I found it useful when implementing the _ticksToDateTimeHelper method, just to hold the result data. It also simplifies the implementation of 'toString' as all the computations needed to get the year, month, day etc. are performed only once and the result is stored in this 'structure'. 
It could also be useful in holding the results of current date/time returned by Time class, as I'm not sure if I can use 'DateTime' module directly in the 'sdk/os/Time' class. Time could simply return this 'raw' date/time data so we could use it in DateTime class.